tests/dummy/app/templates/docs/scroll-into-view.md

```diff
@@ -19,6 +19,35 @@ You should use this modifier whenever you need to have an element scrolled into
 `shouldScroll` can be either a Boolean or a Promise that resolves to a truthy or falsy value. It does not handle a rejected Promise.
 
 
+## Testing
+`scroll-into-view-mock` provides a function that will mock the native browser `scrollIntoView` and allow testing which elements invoked the modifier
+
+`mockScrollIntoView()` - will mock the native API and return an object with the following 2 functions
+* `scrollIntoViewCalledWith(Element|DOM selector)` - tests if the modifier was invoked on the element
+* `resetMock()` - restores the native scrollIntoView function
+
+```javascript
+import mockScrollIntoView from 'ember-scroll-modifiers/test-support/scroll-into-view-mock';
+...
+hooks.beforeEach(function () {
+  this.mockHelperFunctions = mockScrollIntoView();
+});
+
+hooks.afterEach(function () {
+  this.mockHelperFunctions.resetMock();
+});
+...
+function('test scroll into view', () => {
+  ...
+  await render(
+    hbs`<div {{scroll-into-view shouldScroll=true}} data-test-scroll-into-view-selector></div>`
+  );
+  ...
+  this.mockHelperFunctions.scrollIntoViewCalledWith('[data-test-scroll-into-view-selector]');
```

Comment:
Let's wrap this in an `assert.ok` with a descriptor to show how to ideally use the result.